packages/selinux-policy/rules.cil

```diff
@@ -179,12 +179,16 @@
 (allow all_s ephemeral_o (files (mutate mount)))
 
 ; Trusted subjects are allowed to write to and manage mounts
-; for "dynamic" files in /etc.
-(allow trusted_s dynamic_o (files (mutate mount)))
+; for files in /etc.
+(allow trusted_s etc_t (files (mutate mount)))
 
 ; wicked calls netdog which writes /etc/resolv.conf.
 (allow network_t etc_t (files (mutate)))
 
+; Privileged subjects are allowed to write to and manage mounts for
+; the binfmt_misc filesystem.
+(allow privileged_s binfmt_misc_fs_t (files (mutate mount)))
```

Comment:
This is the main change; allowing privileged containers write access to binfmt_misc files.

(Unprivileged containers are restricted by the rule immediately below this one.)